doc/examples/applications/plot_3d_structure_tensor.py

```diff
@@ -188,14 +189,18 @@
 # tissue), especially in the Y-Z plane (``longitudinal=1``).
 
 subplots = np.dstack((sample[:, coords[2], :], sample[:, :, coords[3]]))
-px.imshow(
+fig3 = px.imshow(
     subplots,
     zmin=v_min,
     zmax=v_max,
     facet_col=2,
     labels={'facet_col': 'longitudinal'}
 )
 
+plotly.io.show(fig1)
```

Comment:
This call needs to happen right after `fig1 = ...`. Otherwise only `fig3` is showing: https://2292-2014929-gh.circle-artifacts.com/0/doc/build/html/auto_examples/applications/plot_3d_structure_tensor.html#sphx-glr-auto-examples-applications-plot-3d-structure-tensor-py

Reply:
Oh, thanks. I had only run from the command line and forgot to check the built docs.